microsoft/testsuites/dpdk/dpdkutil.py

```diff
@@ -72,33 +71,9 @@ def __init__(self, _node: Node, _testpmd: DpdkTestpmd) -> None:
         self.vf_hotplug_regex = re.compile(
             f"{test_nic.upper}: Data path switched to VF:"
         )
-        self.vf_slot_removal_regex = re.compile(f"{test_nic.upper}: VF unregistering:")
-
-    def wait_for_dmesg_output(self, wait_for: str, timeout: int) -> bool:
-        search_pattern = None
-        if wait_for == "AN_DISABLE":
-            search_pattern = self.vf_slot_removal_regex
-        elif wait_for == "AN_REENABLE":
-            search_pattern = self.vf_hotplug_regex
-        else:
-            raise LisaException(
-                "Unknown search pattern specified in "
-                "DpdkTestResources:wait_for_dmesg_output"
-            )
-
-        self.node.log.info(search_pattern.pattern)
-        timer = perf_timer.Timer()
-        while timer.elapsed(stop=False) < timeout:
-            output = self.dmesg.get_output(force_run=True)
-            if search_pattern.search(output.replace(self._last_dmesg, "")):
-                self._last_dmesg = output  # save old output to filter next time
-                self.node.log.info(
-                    f"Found VF hotplug info after {timer.elapsed()} seconds"
-                )
-                return True
-            else:
-                time.sleep(1)
-        return False
+        self.vf_slot_removal_regex = re.compile(
+            f"{test_nic.upper}: Data path switched from VF"
+        )
```

Comment:
above two patterns are not used?

Reply:
Ah good catch. I skipped a commit while cherry-picking.